README.md

```diff
@@ -14,257 +14,32 @@ pgvecto.rs is a Postgres extension that provides vector similarity search functi
 ## Why use pgvecto.rs
 
 - 💃 **Easy to use**: pgvecto.rs is a Postgres extension, which means that you can use it directly within your existing database. This makes it easy to integrate into your existing workflows and applications.
+- 🔗 **Async indexing**: pgvecto.rs's index is asynchronously constructed by the background threads and does not block insertions and always ready for new queries.
```

Comment:
Could we add some numbers to show the benefit?

Reply:
we'll have a seperate PR for all the benchmarks including insert/index/query performance